SchoolXplorer.Application/Dtos/CreateSchoolDistrictDto.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace SchoolXplorer.Application.Dtos
+{
+	public class CreateSchoolDistrictDto
```

Comment:
Can this be record? Record struct or something else?

Reply:
converted to record type.